### DISCUSSION TITLE
Cannot import root module

### DISCUSSION THREAD

**Discussion by TomzBench** [Microsoft/node-pty#623] [Q&A] [answered]
## Environment details
os: linux
node-pty: 1.0.0
node: 18.16.1
typescript:  5.1.3

### Does not work
```ts
import pty from "node-pty"

pty.spawn(/* ... */); // Error:  spawn does not exist on type "undefined"
```

### Works
```ts
import {spawn} from "node-pty";

spawn(/* ... */); // works
```

Not sure if this is operator error and im doing something dumb, or maybe the `.t.ds` file does not match what is expected.  in either case the docs don't show how to import so perhaps this is then this issue is more a recommendation to add some notes on the usage in the docs

(Since it compiles via typescript, but fails on runtime with undefined error, it might be a typescript declaration file issue)

**Comment by Tyriar** [comment, marked as answer]
You want to use `import * as pty from 'node-pty';` as you want to import everything as `pty`. `import pty from "node-pty"` imports the default module which doesn't exist. I believe we're doing the right thing here as there's no `export default` in node-pty.d.ts.

**Reply by TomzBench** [reply]
ah.  i see.  The examples look updated too! thanks!
